ubuntu/ubuntu-20.x/ubuntu-20.04-hpc/install.sh

```diff
@@ -12,31 +22,32 @@ $UBUNTU_COMMON_DIR/install_lustre_client.sh
 
 # install mellanox ofed
 ./install_mellanoxofed.sh
+# install nvidia gpu driver
```

Comment:
Nitpick: could you add a blank line ahead of the comment here and the one just below here?

Reply:
I have added the extra blank lines for consistency to install.sh for ubuntu 20.04